src/data/weather.ts

```diff
@@ -77,13 +77,29 @@ const getWindBearingText = (degree: string): string => {
   return degree;
 };
 
-export const getWindBearing = (bearing: string): string => {
+const getWindBearing = (bearing: string): string => {
   if (bearing != null) {
     return getWindBearingText(bearing);
   }
   return "";
 };
 
+export const getWind = (
+  hass: HomeAssistant,
+  speed: string,
+  bearing: string
+): string => {
+  if (bearing != null) {
+    const cardinalDirection = getWindBearing(bearing);
+    return `${speed} ${getWeatherUnit(hass!, "length")}/h (${
+      hass.localize(
+        `ui.card.weather.cardinal_direction.${cardinalDirection.toLowerCase()}`
+      ) || cardinalDirection
+    })`;
+  }
+  return `${speed} ${getWeatherUnit(hass!, "length")}/h`;
```

Comment:
Should the `/h` be here? it's in the getWeatherUnit function right?

Reply:
It seems weird to mee too, but I don't modify this methods I only move them to weather.ts.

Reply:
hmm its because we are using the length unit here. should it be windspeed?

Reply:
You are right. Just below in the getWeatherUnit, the wind_speed case do the same. I change it to wind_speed and remove the /h.